docs/error-reporting/platform-integrations/unity/setup.md

```diff
@@ -46,6 +46,10 @@ Native Crashes on PlayStation and Nintendo Switch are captured via Backtrace Dat
 Offline database capabilities are currently not supported for Nintendo Switch.
 :::
 
+:::note
+The iOS SDK contains a privacy manifest to declare the types of data accessed on the device. Please refer to this [source document](https://github.com/backtrace-labs/backtrace-cocoa/blob/master/PrivacyInfo.xcprivacy) for the specific types of data collected by the SDK.
```

Comment:
hey I recommend to use the unity link instead. In the future those manifest can change

Reply:
Excellent. Updated.